tests/unit/lms/services/canvas_api/conftest.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import sentinel
+
+import pytest
+
+from lms.services.canvas_api import AuthenticatedClient, BasicClient, TokenStore
+from tests import factories
+
+
+@pytest.fixture
+def token_store(db_session, application_instance, lti_user):
+    return TokenStore(
+        db_session,
+        consumer_key=application_instance.consumer_key,
+        user_id=lti_user.user_id,
+    )
+
+
+@pytest.fixture
+def basic_client():
+    return BasicClient("canvas_host")
+
+
+@pytest.fixture
+def http_session(patch):
+    session = patch("lms.services.canvas_api.basic.Session")
+    session = session()
+
+    def set_response(json_data=None, raw=None, status_code=200):
+        session.send.return_value = factories.requests.Response(
+            json_data=json_data, raw=raw, status_code=status_code
+        )
+
+    session.set_response = set_response
+
+    return session
```

Comment:
You decided to mock requests instead of using httpretty?

Reply:
Not in this PR, but yep:

 * Mostly because it's what was already there
 * Lets you follow our standard stuff with mock objects and assertions: e.g.
    * Raising errors
    * Making assertions about arguments passed to send, but don't necessarily make it to a response (like timeout, stream etc.)